engine/flink/minicluster/src/main/scala/pl/touk/nussknacker/engine/flink/minicluster/scenariotesting/FlinkMiniClusterScenarioStateVerifier.scala

```diff
@@ -35,36 +42,47 @@ class FlinkMiniClusterScenarioStateVerifier(
       processVersion: ProcessVersion,
       scenario: CanonicalProcess,
       savepointPath: String
-  ): Try[Unit] = {
+  ): Future[Unit] = {
     legacyFallbackToSingleUseMiniClusterHandler.withSharedOrSingleUseCluster(sharedMiniClusterServicesOpt, scenario) {
       miniClusterWithServices =>
         val scenarioWithOverrodeParallelism = sharedMiniClusterServicesOpt
```

Comment:
scenarioWithOverrodeParallelism
```suggestion
        val scenarioWithOverridenParallelism = sharedMiniClusterServicesOpt
```

Reply:
Good point, but overridden, changed